cpovc_forms/views.py

```diff
@@ -8738,13 +8738,13 @@ def new_cpara(request, id):
         geo_wards = None
     if all_geos_county:
         geo_county = ', '.join(all_geos_county)
-    child.geo_wards = geo_wards
```

Comment:
Changing from child.geo_wards to ge_wards in all areas in the view is causing breaking changes.